apps/balancer-tools/src/app/apr/round/(components)/PoolListTable.tsx

```diff
@@ -29,30 +29,36 @@ import { formatNumber } from "#/utils/formatNumber";
 import { PoolTypeEnum } from "../../(utils)/calculatePoolStats";
 import { formatAPR, formatTVL } from "../../(utils)/formatPoolStats";
 import getFilteredRoundApiUrl from "../../(utils)/getFilteredApiUrl";
+import { formatDateToMMDDYYYY } from "../../api/(utils)/date";
 import { PoolStatsData, PoolStatsResults, PoolTokens } from "../../api/route";
 import { MoreFiltersButton } from "./MoreFiltersButton";
 import { TokenFilterInput } from "./TokenFilterInput";
 
 export function PoolListTable({
-  roundId,
+  startAt,
+  endAt,
   initialData,
 }: {
-  roundId: string;
-  initialData: PoolStatsData[];
+  startAt: Date;
+  endAt: Date;
+  initialData: { [key: string]: PoolStatsData[] };
 }) {
   const pathname = usePathname();
   const searchParams = useSearchParams();
-  const [tableData, setTableData] = useState(initialData);
+  const selectedDate = initialData[formatDateToMMDDYYYY(endAt)];
+
+  const [tableData, setTableData] = useState(selectedDate);
   const [isLoadingMore, setIsLoadingMore] = useState(false);
   const [hasMorePools, setHasMorePools] = useState(true);
 
   useEffect(() => {
-    setTableData(initialData);
-    setHasMorePools(!(initialData.length < 10));
+    setTableData(initialData[formatDateToMMDDYYYY(endAt)]);
+    setHasMorePools(!(initialData[formatDateToMMDDYYYY(endAt)].length < 10));
```

Comment:
Will be changed on #BAL-809

Reply:
should we leave a `TODO` with the task reference to catch that?